composer.json

```diff
@@ -4,7 +4,7 @@
     "license": "MIT",
     "description": "A sentry client for Flow and Neos",
     "require": {
-        "typo3/flow": ">=2.3.9 <=2.3.12 || >=3.0.3",
+        "typo3/flow": "~2.3.9 || ~3.0.3",
```

Comment:
`~3.0.3` will translate to `>=3.0.3 <3.1.0` which would not work for Flow versions `3.2` and `3.3`. I'd suggest to change this to `^3.0.3` which would mean `>=3.0.3 < 4.0.0` which should fit the package compatibility.